.github/workflows/package-fedora.yml

```diff
@@ -0,0 +1,38 @@
+name: Package GEDKeeper3 for Fedora
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  DOTNET_VERSION: '6.0.*'
+
+jobs:
+  build-fedora:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Init
+      uses: actions/checkout@v3
+      with:
+        submodules: recursive
+    - name: .NET SDK check
+      run: dotnet sdk check
+# error NETSDK1084: There is no application host available for the specified RuntimeIdentifier 'freebsd-x64'
+# https://learn.microsoft.com/en-us/dotnet/devops/dotnet-build-github-action
```

Comment:
Yes, RID `freebsd-x64` should have been removed a long time ago. I seem to have taken it from the freebsd dotnet build articles, but they have long-standing problems with MS (it seems) and the RID is not valid.